vsphere/tests/conftest.py

```diff
@@ -37,13 +42,17 @@ def historical_instance():
 
 @pytest.fixture
 def mock_type():
```

Comment:
What's the purpose of `mock_type` ?

Could you add a comment ?

Reply:
Oh sure